src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$sizeBlocks: 300px;
+$itemColors:
+  "red" #f00,
+  "red80" #c00,
+  "red60" #900,
+  "red40" #600,
+  "red20" #300,
+  "black" #000;
```

Comment:
I think you can achieve the same effect without modifier classes, but just with each block index